params.go

```diff
@@ -73,6 +82,36 @@ var (
 		0.4,
 		"Scaling parameter used in the population model. Larger coeff => deer pop. regenerates faster.",
 	)
+	foragingMaxFishPerHunt = flag.Uint(
```

Comment:
was wondering if we should change to `foraging<Animal>xxx`? similar for deer

Reply:
also, thank you again for reading existing code!

Reply:
Sorry don't quite understand the suggestion? @lhl2617

Reply:
to make it easier to find configs the naming should be `foragingDeerMaxPopulation` so we can just do a quick search :) (we'd currently miss out foragingMaxDeer)